jwql/website/apps/jwql/views.py

```diff
@@ -746,6 +747,50 @@ def jwqldb_table_viewer(request, tablename_param=None):
     return render(request, template, context)
 
 
+def log_view(request):
+    """Access JWQL monitoring logs from the web app.
+
+    Parameters
+    ----------
+    request : HttpRequest object
+        Incoming request from the webpage
+
+    Returns
+    -------
+    HttpResponse object
+        Outgoing response sent to the webpage
+    """
+
+    template = 'log_view.html'
+    log_path = get_config()['log_dir']
+    log_name = request.POST.get('log_submit', None)
+
+    full_uri = request.build_absolute_uri()
+
+    if 'dljwql' in full_uri:
```

Comment:
Do you think this is something that would be useful to have as its own general function somewhere, or is there not enough use for it to warrant a separate function?

Reply:
So this was causing the issue with the production logs being show on the test server. I went ahead and changed it to use the `socket` library to get the hostname of the machine rather than the URL.